### ISSUE TITLE
style tag should not be required

### ISSUE THREAD

**Issue by gKreator** [icanjs/steal-vue#2]
I get an error when omitting the style tag in a .vue file

![image](https://user-images.githubusercontent.com/1431487/27410878-fbfa6c30-56af-11e7-8728-ed091d25adb4.png)

![image](https://user-images.githubusercontent.com/1431487/27410914-3c762402-56b0-11e7-9ae1-f49967f8e000.png)

**Comment by gKreator** [comment]
I fixed this error on steal-vue.js with this and all of the styles work properly.
![image](https://user-images.githubusercontent.com/1431487/27412447-52e8cfbe-56ba-11e7-96e1-b76e60f46c0e.png)

I do get an empty style tag in the inspector:
![image](https://user-images.githubusercontent.com/1431487/27413941-f53f72fa-56c3-11e7-974d-0636e5f69627.png)

**Comment by marshallswain** [comment]
@gKreator care to make a PR?  Even with an empty style tag, it's an improvement over the current behavior.

**Comment by marshallswain** [comment]
Also, thank you for pointing this out!

**Comment by gKreator** [comment]
Yup I will do so when I get home tonight. I just learned that both the template tag and the script tag are not required. I know that at least the removal of the script tag causes a fatal error.  I did not try removing the template tag, but I am guessing that it will crash as well. If you could point me in the right direction I can try and fix some of this. I have never worked on a steal plugin before.

**Comment by marshallswain** [comment]
The `script` tag will probably have to be required.  We should throw a proper error if there's no script because the current implementation depends on it.  For the template, I think it just needs to conditionally inject the template string, here: https://github.com/icanjs/steal-vue/blob/master/steal-vue.js#L49

**Comment by gKreator** [comment]
![image](https://user-images.githubusercontent.com/1431487/27460350-ed38db36-5778-11e7-86a1-32cfde067c9b.png)

This is better, I guess you do need a template tag, but now we get a proper error.

![image](https://user-images.githubusercontent.com/1431487/27460358-f63ad482-5778-11e7-9619-c050ca349d5d.png)

I will try to fix the other build issue and then I will post the code.
